Source/Core/Core/ActionReplay.cpp

```diff
@@ -134,20 +133,22 @@ void AddCode(ARCode code)
   }
 }
 
-void LoadAndApplyCodes(const IniFile& global_ini, const IniFile& local_ini)
+void LoadAndApplyCodes(Config::Layer& global_ini, Config::Layer& local_ini)
```

Comment:
LoadCodes() takes a non-const Layer (because it uses GetOrCreateSection), so I'm not sure I can change this to a const reference.